04_setup_ironic.sh

```diff
@@ -12,7 +12,7 @@ source ocp_install_env.sh
 # with "-" converted to "_" e.g. to use a custom ironic-inspector
 #export IRONIC_INSPECTOR_LOCAL_IMAGE=https://github.com/metal3-io/ironic-inspector-image
 #export IRONIC_RHCOS_DOWNLOADER_LOCAL_IMAGE=https://github.com/openshift-metal3/rhcos-downloader
-#export BAREMETAL_OPERATOR_LOCAL_IMAGE=192.168.111.1:5000/localimages/bmo:latest
+#export BAREMETAL_OPERATOR_LOCAL_IMAGE=$LOCAL_REGISTRY_ADDRESS/localimages/bmo:latest
```

Comment:
This is just an example comment showing how to set this in config_$USER.sh, where common.sh isn't necessarily souurced, so maybe we should leave this one as-is?

Reply:
Done